torchaudio/compliance/kaldi.py

```diff
@@ -0,0 +1,209 @@
+import math
+import random
+import torch
+
+
+__all__ = [
+    'spectrogram'
+]
+
+# numeric_limits<float>::epsilon()
+EPSILON = torch.tensor(1.19209290e-07, dtype=torch.get_default_dtype())
```

Comment:
drive by comment: you can use `torch.finfo(torch.get_default_dtype()).eps`

Reply:
@gqchen, indeed, good catch. And it give the same value as cpp numeric_limits<float>::epsilon() so it will be giving same results as kaldi
>>> torch.finfo(torch.get_default_dtype()).eps
1.1920928955078125e-07